hoomd/hpmc/pytest/test_shape.py

```diff
@@ -3,40 +3,41 @@
 from hoomd.hpmc import _hpmc
 import numpy as np
 import pytest
+from hoomd.hpmc.pytest.conftest import *
+
+
+def check_dict(shape_dict, args):
+    for key, val in args.items():
+        if isinstance(shape_dict[key], list) and len(shape_dict[key]) > 0 \
+           and key != 'shapes':
+            np.testing.assert_allclose(shape_dict[key], val)
+        elif key == 'shapes':
+            for shape_args in shape_dict[key]:
+                for shape_key, shape_val in shape_args.items():
+                    if isinstance(shape_args[shape_key], list) \
+                       and len(shape_args[shape_key]) > 0:
+                        np.testing.assert_allclose(shape_args[shape_key],
```

Comment:
Isn't this just testing that `shape_args[key] == shape_args[key]`. At this point we aren't using the passed val from `args` at all.

Reply:
Just a marker that this has not be addressed yet.

Reply:
Fixed a25e2c3e